src/ui/server/helpers/date/choose-date-format/index.ts

```diff
@@ -0,0 +1,12 @@
+import { format } from 'date-fns';
+
+/**
+ * chooseDateFormat
+ * Format a date with whichever pattern chosen
+ * @param {Date} timestamp
+ * @param {String} pattern pattern of format
+ * @returns {String} Formatted date
+ */
+const chooseDateFormat = (timestamp: Date, pattern: string): string => format(new Date(timestamp), pattern);
+
+export default chooseDateFormat;
```

Comment:
We already have a `formatDate` helper function - `helpers/date/format-date`

I think we should update that so it has an optional `format` param, and if none is provided it defaults to whats currently in there `'d MMMM yyyy'`